src/vsgXchangeConfig.cmake.in

```diff
@@ -2,6 +2,7 @@ include(CMakeFindDependencyMacro)
 
 find_dependency(Vulkan)
 find_dependency(vsg)
-@FIND_DEPENDENCY@
+@FIND_DEPENDENCY_ASSIMP@
```

Comment:
The change to using FIND_DEPENDENCY_ASSIMP and _KTX rather than added additional dependencies to a FIND_DEPENDENCY var results in the OSG dependency from being added.

The include(osg/build_vars.cmake) wraps up all the OSG related vars.  The original scheme might not be a perfect but with additional functionality would at least need to avoid breaking existing functionality.

So with we'll need to introduce a FIND_DEPENDENCY_OSG or have all these dependencies add themselves to FIND_DEPENDENCY.

Reply:
When using FIND_DEPENDENCY, the resulted file looks like this:

`
find_dependency(assimp); find_dependency(ktx)
`

but it should be

`find_dependency(assimp)`
`find_dependency(ktx)`

That was the idea behind my changes.